lib/clearance/session.rb

```diff
@@ -81,7 +81,15 @@ def sign_out
       end
 
       @current_user = nil
-      cookies.delete remember_token_cookie
+
+      if Clearance.configuration.cookie_domain
+        cookies.delete(
+          remember_token_cookie,
+          domain: Clearance.configuration.cookie_domain,
```

Comment:
Test case confirms that we cannot simply delete with `domain: nil` and must not specify the cookie option all together for cases without a custom domain.